POKITTO_CORE/PokittoCore.cpp

```diff
@@ -312,6 +312,43 @@ void Core::titleScreen(const char*  name, const uint8_t *logo){
 	}
 }
 
```

Comment:
The parameter useDirectMode is a default parameter, so it does not break the compatibility